interface/wx/maskededit.h

```diff
@@ -0,0 +1,935 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        maskededit.h
+// Purpose:     interface of wxMaskedEdit
+// Author:      wxWidgets team
+// RCS-ID:      $Id$
+// Licence:     wxWindows license
+/////////////////////////////////////////////////////////////////////////////
+
+/**
+@defgroup group_maskededit_predef Masked edit predefined functions
+*/
+
+/**
+    Struct with the colours used for different cases with wxMaskedEditText and
+    wxMaskedEditCombo.
+
+    If you don't set a member of this struct, the standard system default
+    colour for the control is used for it.
+
+    @library{wxcore}
+    @category{ctrl}
+
+    @see wxMaskedEditText, wxMaskedEditCombo
+
+    @since 3.1.6
+
+*/
+struct wxMaskedEditColours
+{
+    /// Foreground colour used when everything is OK.
+    wxColour colOKForegn;
+    /// Background colour used when everything is OK.
+    wxColour colOKBackgn;
```

Comment:
Naming here is very different from wx standard. We use `Fg` and `Bg` and I've never seen "foreground" abbreviated like this, not only inside wx. IOW these fields should be renamed to `colOK{Fg,Bg}` or, for more clarity, `colOK{Fore,Back}ground`.